tests/utils/throttleTick.test.js

```diff
@@ -0,0 +1,77 @@
+/* global assert, suite, test */
+var throttleTick = require('utils').throttleTick;
+
+suite('utils.throttleTick', function () {
+  var ts;
```

Comment:
rather than keeping track with arrays, you can use [sinon test spies](http://sinonjs.org/docs/#sinonspy): `var throttleSpy = this.sinon.spy(utils, 'throttleTick'`)` to assert how many times it was called, and its arguments.